### PR TITLE
Fix mtls test env run script.

### PR DESCRIPTION
**Pull request by rigelbm**
Missing packaging commands.

### CODE REVIEW

File: testing/environments/mtls/run.sh [hunk 1]
```diff
@@ -42,6 +42,9 @@ docker pull 368821881613.dkr.ecr.us-west-2.amazonaws.com/confluentinc/cp-zookeep
 # Download latest cp-server image.
 docker pull 368821881613.dkr.ecr.us-west-2.amazonaws.com/confluentinc/cp-server:master-latest
 
+# Make sure kafka-rest is packaged.
+mvn -f "$base_dir"/kafka-rest/pom.xml -Dmaven.test.skip=true clean package
```
Comment (dimitarndimitrov) [inline testing/environments/mtls/run.sh:46]: nit: Do we want to skip compiling tests?
`-DskipTests` will skip running the tests while still compiling them, and will do that only negligibly slower than `-Dmaven.test.skip=true`.

**Review by dimitarndimitrov — APPROVED** [review]
+1.

P.S. Are the ports for the Kafka brokers correct (see below). The HTTP ones seem to be wrong.
```
  * 3 Kafka brokers:
    1. `kafka-1:9291`, `http://kafka-1:9193`
    2. `kafka-2:9292`, `http://kafka-2:9293`
    3. `kafka-3:9293`, `http://kafka-3:9393`
```

**Review by mtodzo — APPROVED** [review]
thanks again!

File: testing/environments/mtls/run.sh (hunk 1 above)
Comment (rigelbm) [reply testing/environments/mtls/run.sh:46]: Done.

**Comment by dimitarndimitrov** [comment]
@rigelbm FWIW, the `mtls` and `sasl_plain` environments still use `-Dmaven.test.skip=true` (not that I mind too much).

**Pull request merged**